_layouts/index.html

```diff
@@ -18,32 +18,53 @@ <h1 class="logo">Helm</h1>
       <div id="boat"></div>
     </section>
 
+    <section class="panel -blue" data-purpose="news-and-updates">
+      <div class="news-and-updates">
+        <div class="news-and-updates__item container group">
+          <h2>Helm comes in two great flavors!</h2>
+          <article class="console">
+            <p>
+              The teams at Deis and Google are working hard to bring you a new and improved Helm.
+              In the meanwhile, since so many in the Kubernetes community have come to depend on the
```

Comment:
Style comment, so as you please. I'd shorten this to "The Deis team is committed to maintaining the original tool as 'Helm Classic' until the new Helm serves as a viable replacement--new name, same great taste!".